optional_plugins/html/avocado_result_html/__init__.py

```diff
@@ -267,6 +267,8 @@ def _render(self, result, output_path):
             report_file.write(report_contents)
 
     def render(self, result, job):
+        if job.status == "RUNNING":
```

Comment:
This is currently a duplicated safeguard.  I'm looking further along in the code to check why it's necessary.